### PR TITLE
Add KISSFC target

### PR DESCRIPTION
**Pull request by iioel**
Add support of KissFC v1

### CODE REVIEW

File: src/main/target/KISSFC/target.h [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+#define AVOID_UART2_FOR_PWM_PPM
+
+#define TARGET_IO_PORTA         0xffff
+#define TARGET_IO_PORTB         0xffff
+#define TARGET_IO_PORTC         0xffff
+#define TARGET_IO_PORTD         0xffff
+#define TARGET_IO_PORTF         (BIT(4))
+
+#define USABLE_TIMER_CHANNEL_COUNT 11
```
Comment (digitalentity) [inline src/main/target/KISSFC/target.h]: This is not needed for INAV

File: src/main/target/KISSFC/target.h [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+#define USED_TIMERS             (TIM_N(1) | TIM_N(2) | TIM_N(3) | TIM_N(4) | TIM_N(8) | TIM_N(15) | TIM_N(16) | TIM_N(17))
```
Comment (digitalentity) [inline src/main/target/KISSFC/target.h]: This is not needed

**Review by digitalentity — CHANGES_REQUESTED** [review]
Please rebase your changes on development branch and verify the `target.h` definitions

**Comment by iioel** [comment]
Useless lines removed and I have rebased my branch. Is it ok ?

**Comment by iioel** [comment]
Timers reviewed.
Just need to test on the board.

**Review by digitalentity — APPROVED** [review]

**Comment by iioel** [comment]
Motors and servo output are working now (tested on the board), reciver with crsf too

**Pull request merged**